utils/metrics/src/main/java/org/apache/edgent/metrics/MetricsSetup.java

```diff
@@ -111,6 +144,25 @@ private MBeanServer mbeanServer() {
         }
         return mBeanServer;
     }
+
+    /**
+     * Creates a default directory for metrics.
+     * 
+     * @return directory.getPath()
+     */
+    private String createDefaultDirectory() {
+        Path currentRelativePath = Paths.get("");
+        String pathMetrics = currentRelativePath.toAbsolutePath().toString() + FOLDER_METRICS;
+        File directory = new File(pathMetrics);
+        
+        if (!directory.mkdirs()) {
+            // Log: "Could not create the directory log"
```

Comment:
I'm going to merge this now.  Thomas, when you get a chance it would be great to add the logging.  Thanks!